### PR TITLE
migrate code from googleapis/nodejs-ai-platform

### PR DESCRIPTION
**Pull request by pattishin**
- feat!: initial generation of library (#4)
- chore: release 1.0.0 (#6)
- feat: adds enhancements to library (#22)
- chore: release 1.1.0 (#29)
- feat: adds cleaner utility for orphaned resources (#34)
- chore: release 1.2.0 (#36)
- feat(samples): adds samples for enhanced version of library (#16)
- Remove cleaning script from tests (#58)
- chore: release 1.3.0 (#39)
- build: increase retries and timeout for flaky tests (#60)
- build: increase retries for #33 (#61)
- samples: fix region tag (#65)
- fix: reduces image object detection test file size (#69)
- feat(samples): add additional samples to library (#25)
- feat: updates enhanced client library veneer and samples (#73)
- chore: release 1.4.0 (#64)
- test: extend timeout (#78)
- chore: release 1.5.0 (#79)
- Rename apiEndpoint for samples (#82)
- chore: release 1.6.0 (#87)
- Fix typo in path (#112)
- samples: adds batch prediction samples for text (#119)
- chore: new owl bot post processor docker image (#121)
- chore: release 1.7.0 (#125)
- samples: adds missing Node.js samples (#128)
- chore: release 1.7.1 (#144)
- test: deploy mode test failure (#145)
- chore: release 1.7.2 (#153)
- chore: release 1.7.3 (#159)
- chore: release 1.7.4 (#165)
- chore: release 1.8.0 (#169)
- chore: release 1.9.0 (#171)
- chore: release 1.9.1 (#173)
- chore: release 1.10.0 (#179)
- chore: release 1.10.1 (#185)
- feat: add prediction service RPC RawPredict to aiplatform_v1beta1 feat: add tensorboard service RPCs to aiplatform_v1beta1: BatchCreateTensorboardRuns, BatchCreateTensorboardTimeSeries, WriteTensorboardExperimentData feat: add model_deployment_monitori... (#196)
- chore: release 1.11.0 (#194)
- samples: remove unnecessary back-ticks from samples    (#209)
- test: added back BQ tabular dataset sample test (#215)
- chore: release 1.12.0 (#208)
- chore: release 1.13.0 (#230)
- docs: Added eval_id comments to node.js samples (#236)
- chore: add '_sample' suffix to samples (#242)
- chore: release 1.14.0 (#235)
- fix: remove duplicate sample (#243)
- chore(main): release 1.15.0 (#263)
- chore(main): release 1.16.0 (#270)
- chore(main): release 1.17.0 (#276)
- chore(main): release 1.18.0 (#279)
- chore(main): release 1.19.0 (#281)
- fix: added retries to flaky test (#299)
- build!: update library to use Node 12 (#304)
- feat(samples): add create-featurestore samples (#317)
- chore(main): release 2.0.0 (#305)
- chore(main): release 2.1.0 (#345)
- feat(samples): add remaining featurestore api samples (#338)
- feat(samples): add entity type apis samples (#339)
- feat(samples): add feature apis samples (#340)
- feat(samples): add feature values apis samples (#341)
- chore(main): release 2.2.0 (#348)
- chore(deps): update dependency uuid to v9 (#367)
- chore(main): release 2.3.0 (#369)

### CODE REVIEW

**Review by kweinmeister — APPROVED** [review]

**Comment by NimJay** [comment]
I created a test pull-request at https://github.com/googleapis/nodejs-ai-platform/pull/469 (in the original repo).
I wanted to double check whether these tests were already failing in the original repo (`googleapis/nodejs-ai-platform`).
More specifically the test if for `create-featurestore-fixed-nodes-sample.test.js`.
`create-featurestore-fixed-nodes-sample.test.js` is passing in the original repo.

This confirms that we're just missing some CI environment setup in `GoogleCloudPlatform/nodejs-docs-samples` that we already have inside `googleapis/nodejs-ai-platform`.

**Comment by pattishin** [comment]
> I created a test pull-request at [googleapis/nodejs-ai-platform#469](https://github.com/googleapis/nodejs-ai-platform/pull/469) (in the original repo). I wanted to double check whether these tests were already failing in the original repo (`googleapis/nodejs-ai-platform`). More specifically, I'm running the `create-featurestore-fixed-nodes-sample.test.js` tests. `create-featurestore-fixed-nodes-sample.test.js` [is passing](https://source.cloud.google.com/results/invocations/09fa5992-c826-4b8b-a6fe-441194a424e7/targets/cloud-devrel%2Fclient-libraries%2Fnodejs%2Fpresubmit%2Fgoogleapis%2Fnodejs-ai-platform%2Fnode12%2Fsamples-test/log) in the original repo.
> 
> This confirms that we're just missing some CI environment setup in `GoogleCloudPlatform/nodejs-docs-samples` that we already have inside `googleapis/nodejs-ai-platform`.

Thanks Nim! Currently updating this as we speak 👍🏼 I'll put a WIP in the PR title

**Comment by pattishin** [comment]
All ready for another look 👓

File: ai-platform/snippets/quickstart.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * TODO: add an actual quickstart example.
+ */
+async function main() {
+  // [START aiplatform_quickstart_sample]
```
Comment (ace-n) [inline ai-platform/snippets/quickstart.js:21]: Is this used at all? I tried searching for this region tag, but couldn't find anything relevant. 😞

**Review by ace-n — APPROVED** [review]
One question about the quickstart, otherwise LGTM. 👍

File: ai-platform/snippets/quickstart.js (hunk 1 above)
Comment (pattishin) [reply ai-platform/snippets/quickstart.js:21]: @ace-n For this particular sample set, we're migrating all even if they are not used yet.

**Pull request merged**